### DISCUSSION TITLE
Release do RocketChat !

### DISCUSSION THREAD

**Discussion by TI-GRUPO-PEGASUS** [RocketChat/Rocket.Chat#30100] [Q&A] [answered]
Good morning !
I would like to validate whether in this new release of RocketChat, there is the question of whether we are viewing whether or not the user has read the message sent, or whether this function is only available in the paid version of RocketChat.

**Comment by ggazzo** [comment, marked as answer]
the read receipts feature is enterprise only

**Reply by TI-GRUPO-PEGASUS** [reply]
Boa tarde !

Guilherme, obrigado pela resposta.
